news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title:  "PR Testing on PrestaShop"
+subtitle: ""
```

Comment:
```suggestion
subtitle: "How the QA team handle the PR testing ?"
```
Could you complete the subtitle ?
It will be displayed here:
![capture d'écran_2890](https://user-images.githubusercontent.com/13449658/101608085-9ab91380-3a05-11eb-95a4-85013e20de4e.png)

Reply:
Thanks (didn't know where it was used :sweat_smile:  ), I added a subtitle